packages/http-client-python/generator/pygen/codegen/models/operation_group.py

```diff
@@ -152,6 +152,27 @@ def imports(self, async_mode: bool, **kwargs: Any) -> FileImport:
                 f"{self.client.name}MixinABC",
                 ImportType.LOCAL,
             )
+        else:
+            file_import.add_submodule_import(
+                "" if self.code_model.is_azure_flavor else "runtime",
+                f"{'Async' if async_mode else ''}PipelineClient",
+                ImportType.SDKCORE,
+            )
+            file_import.add_submodule_import(
+                ".._configuration",
+                f"{self.client.name}Configuration",
+                ImportType.LOCAL,
+            )
+            file_import.add_msrest_import(
+              serialize_namespace=kwargs.get("serialize_namespace", self.code_model.namespace),
+              msrest_import_type=MsrestImportType.Serializer,
+              typing_section=TypingSection.REGULAR,
+            )
+            file_import.add_msrest_import(
+              serialize_namespace=kwargs.get("serialize_namespace", self.code_model.namespace),
+              msrest_import_type=MsrestImportType.SerializerDeserializer,
```

Comment:
We need to consider multiapi here like https://github.com/microsoft/typespec/blob/dada04463b75ff1e7a299c8a0b24644fc1fb479a/packages/http-client-python/generator/pygen/codegen/models/imports.py#L276-L279